File: BOJ/CPP/단계별로_풀어보기/02_if문/BOJ_1330.cpp
/*
    BOJ 1330 : 두 수 비교하기

    author : kdgyun
    link https://st-lab.tistory.com
    link https://github.com/kdgyun
    Created by kdgyun
    Copyright © 2021 kdgyun. All rights reserved.
*/

#include <iostream>
using namespace std;

int main(int argc, char const *argv[])
{
    int A, B;
    cin >> A >> B;
    cout << ((A > B) ? ">" : (A < B) ? "<" : "==");
    return 0;
}

File: BOJ/Java/단계별로_풀어보기/B_02_조건문/BOJ_1330.java
package 단계별로_풀어보기.B_02_조건문;

/**
 * BOJ 1330 : 두 수 비교하기
 * 
 * @author kdgyun
 * {@link https://st-lab.tistory.com}
 * {@link https://github.com/kdgyun}
 *
 */

import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.io.IOException;
import java.util.StringTokenizer;

public class BOJ_1330 {

	public static void main(String[] args) throws IOException {
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

		StringTokenizer st = new StringTokenizer(br.readLine(), " ");
		
		int A = Integer.parseInt(st.nextToken());
		int B = Integer.parseInt(st.nextToken());

		System.out.println((A > B) ? ">" : ((A < B) ? "<" : "=="));
	}

}
